tests/test_basic.py

```diff
@@ -0,0 +1,223 @@
+
+def test_csv_bbox(script_runner, tmpdir):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/cities_NL.csv', 
+        '--detail', 'bbox')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "[[10,20],[11,50]" in ret.stdout, "bbox is printed to console"
+
+def test_csv_time(script_runner, tmpdir):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/cities_NL.csv', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "temporal extent is from 2017 ==> 2018" in ret.stdout, "time value is printed to console"
+
+def test_csv_time_invalid(script_runner, tmpdir):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/cities_NL.csv', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert not ret.success, "process should return success"
+    assert ret.stderr == 'invalid time format', "stderr should not be empty"
+
+def test_gml_bbox(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/clc_1000_PT.gml', 
+        '--detail', 'bbox')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+
+def test_gml_time(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/clc_1000_PT.gml', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "2018-11-14 00:00:00" in ret.stdout,  "time value is printed to console"
+
+def test_gml_time_invalid(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/clc_1000_PT.gml', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == 'invalid time format', "stderr should not be empty"
+
+def test_shp_bbox(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/Abgrabungen_Kreis_Kleve_Shape.shp', 
+        '--detail', 'bbox')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+
+def test_dbf_bbox(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/Abgrabungen_Kreis_Kleve_Shape.dbf', 
```

Comment:
You do not need to test the `.dbf` file, it belongs to the shape file.

Reply:
Done.